unyt/tests/test_unyt_array.py

```diff
@@ -57,6 +57,7 @@
     UnitOperationError,
     UnitParseError,
     UnitsNotReducible,
+    UnitOperationWarning,
```

Comment:
ordering

Reply:
Saw there were a few others out of order here or incorrectly grouped here too, re-grouped to follow https://www.python.org/dev/peps/pep-0008/#imports